release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1055,6 +1055,29 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* Currently, Operator Lifecycle Manager (OLM) displays pod security admission (PSA) warnings for catalog sources installed outside of the `openshift-marketplace` namespace. 
++
+Workaround: Cluster administrators can remove the warnings by setting PSA labels in namespace YAML files. Apply the `baseline` label to the `warn` and `audit` modes, as shown in the following example:
++
+.Example `manifests/<target_namespace>.yaml` file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+...
+  labels:
+    openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/audit: baseline <1>
+    pod-security.kubernetes.io/warn: baseline <2>
+  name: "openshift-marketplace"
+----
+<1> Set the `audit` mode label to `baseline`.
+<2> Set the `warn` mode label to `baseline`.
++
+For more information, see link:https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces[Pod Security Admission labels for namespaces]. (link: https://bugzilla.redhat.com/show_bug.cgi?id=2100323[*BZ#2100323*])
```

Comment:
/hold

There is some confusion surrounding the bz. Holding this till that gets resolved.